src/engine/xrinput.js

```diff
@@ -142,6 +159,19 @@ State.eventHandler.addEventListener("inputsourceschange", e => {
   XRInput.controllerGrips = [];
   XRInput.inputSources = e.session.inputSources;
 
+  XRInput.inputSources.forEach((inputSource, controllerIndex) => {
+    let controller = Renderer.xr.getController(controllerIndex);
+    let controllerGrip = Renderer.xr.getControllerGrip(controllerIndex);
+
+    if (inputSource.handedness === "left") {
```

Comment:
Is there a special case when left and right are flipped?

Reply:
All these handedness bugs blend together 🤪 @MichaelHazani Do you remember if the handedness bug was fixed at the XRPackage level? I think it mostly occurred on Index controllers.

Reply:
good question, I no longer remember either. Even if it did, it would best be addressed at the chimera level, since SC is more app-agnostic whereas chimera relies on MC, flaws and handedness quirks notwithstanding. In other words, SC can be expected to work just fine under "normal browser" circumstances.